### DISCUSSION TITLE
Issue with Webhook Automation

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#7132] [Help] [answered]
Self Hosted V.1.2.20 on Latest Debian / multi image docker.

Scenario : I have a contact form in Wordpress i want to submit to a budibase app.

I have the form ready in WP.
I create a new Webhook.
Click SetUp Webhook
Copy the URL , update my WP form and send a submission. 
Budibase detects the submission telling me there are 13 bindable values

Next add a Create Row action to the chain.
I choose my (my_sql) table
I set fields to bindings i.e name => {{ trigger.name }}

I click "finish and test"
Once I start entering values into the fields, i get a toast saying "Error saving automation" 
<img width="541" alt="Screenshot 2022-08-07 at 10 03 33 PM" src="https://user-images.githubusercontent.com/80512170/183306993-3d0f1f0f-d787-49ba-8618-0199e08dca81.png">

If you publish after this , the automation doesn't get triggered on calls to the webhook.

this setup used to work btw , already caught a few submissions. 

Any ideas ?

**Comment by melohagan** [comment]
Hey @poirazis 

This may be a temporary blip. If you delete the automation and try again do you see the same issue?

If you try a `budi hosting --stop` followed by a `budi hosting --start` does that help at all?

**Reply by poirazis** [reply]
I did all those.
I even upgraded from 1.2.19 -> 1.2.20, updated npm et all ( see other discussion we had ) and updated and restarted budibase :)

**Reply by melohagan** [reply]
Would you be able to provide the detailed error from the console & network tabs please? I'm also wondering if by any chance using the internal Db instead of the MySQL datasource makes a difference just as a test.

**Reply by poirazis** [reply]
Hey @melohagan ,
under the Console tab the error is "Empty string passed to getElementById().", and there is no error in Network.
does this help at all ?

**Reply by poirazis** [reply]
Hey @melohagan 
I started a new app from scratch (1.2.20) and did the same exercise ( with an internal BB table ) and the same problem appears.

**Reply by melohagan** [reply]
Thanks @poirazis I'll have to investigate this further. I'll get a look tomorrow hopefully!

**Comment by melohagan** [comment, marked as answer]
@poirazis I've been able to reproduce the issue and I've raised a bug here: https://github.com/Budibase/budibase/issues/7206

**Comment by poirazis** [comment]
Any idea when a fix will be rolled out @melohagan ?

**Reply by melohagan** [reply]
The fix has been merged into master, so it just needs to be built and released. This will probably happen some point during the week!

**Reply by poirazis** [reply]
Thank you !
